WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -453,7 +452,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                             return;
                         }
 
-                        updateFeaturedImage(Integer.parseInt(ids.get(0)));
+                        updateFeaturedImage(Long.parseLong(ids.get(0)));
```

Comment:
Having a default value here is going to make weird things (broken or wrong featured image). I prefer to get the exception and crash the app, at least, we'll be alerted something is wrong.